HW03/ekaterina_balabanovich/spec/balabanovich_hw03_task_1_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require_relative '../balabanovich_hw03_task_1'
+
+RSpec.describe 'Task1' do
+  describe '#error_extract' do
+    context 'when one error occurred' do
```

Comment:
We usually use subject for each `describe` block 

```suggestion
  describe '#error_extract' do
    subject { error_extract(logs) }
    
    context 'when one error occurred' do
```

Reply:
And with subject we can use a shorter version of it `it { is_expected.to ...` 
```suggestion
  describe '#error_extract' do
    subject { error_extract(logs) }
    
    context 'when one error occurred' do
    # ...
    it { is_expected.to eq("2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>\n") }
```